pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
+    <span class="edit-link">
+      [<a href="{% url 'pootle-staticpages-edit' pk=page.id page_type='announcements' %}">edit</a>]
```

Comment:
The following concern was raised on the PR initial message:

> Not sure it's right to get the `page_type` this way but couldn't figure out a better way, suggestions welcome.

I don't like the idea of hardcoding this. I would suggest to create a new URL on `staticpages` app named `pootle-staticpages-announcement-edit` that imports `ANN_TYPE` from `staticpages.views` and uses it for the URL pattern. This way it won't be necessary to hardcode anything on the template.

@julen Thoughts on this?